testing/test_package_docs/ex/Deprecated/operator_equals.html

```diff
@@ -66,10 +66,7 @@ <h1 class="title">
   <div class="row">
 
   <div class="col-xs-6 col-sm-3 col-md-2 sidebar sidebar-offcanvas-left">
-    <h5><a href="index.html">test_package</a></h5>
-    <h5><a href="ex/ex-library.html">ex</a></h5>
-    <h5><a href="ex/Deprecated-class.html">Deprecated</a></h5>
-
+    <h5>class Deprecated</h5>
```

Comment:
Was it a conscious choice to drop the link to class even though this is a member page?